### DISCUSSION TITLE
quest for the name of statues/artwork [rejected]

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#2399] [Ideas]
https://github.com/streetcomplete/StreetComplete/issues/2125#issuecomment-749509735

> I like this suggestion! And I've been thinking for a long time about creating a suggestion for a quest for the name of statues/artwork (when they have a plaque with a name), but I've been unsure if this would be of interest to anyone else. Just asking here if you think I should create a new issue with this quest suggestion? :)

**Comment by matkoniecz** [comment]
I think that it depends on how many of them are named. For shops and museums it makes sense to ask as nearly all of them are named.

Not sure is it true for artworks.

From technical side it would be added to https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/place_name/AddPlaceName.kt#L61

**Reply by westnordost** [reply]
In my experience, most are not named, or if they are, there is no plaque for that.
